cfme/v2v/migration_plans.py

```diff
@@ -0,0 +1,573 @@
+
+
+class ArchivedPlansView(MigrationPlanView):
+    TITLE = "Archived"
+    migration_plans_list = MigrationPlansList('plans-archived-list')
+
+    @property
+    def is_displayed(self):
+        in_view = VersionPicker({
+            Version.lowest(): self.in_migration_plan and
+                              self.migr_dropdown.read() == self.TITLE + " Plans",
+            '5.10': self.in_migration_plan and self.TITLE in self.title.read()
+        })
+        return in_view
+
+
+class AddMigrationPlanView(View):
+    title = Text(locator='.//h4[contains(@class,"modal-title")]')
+    name = TextInput(name='name')
+    description = TextInput(name='description')
+    back_btn = Button('Back')
+    next_btn = Button('Next')
+    cancel_btn = Button('Cancel')
+    form_title_text = VersionPicker({
+        Version.lowest(): 'Migration Plan Wizard',
+        '5.10': 'Create Migration Plan'
+    })
+
+    @View.nested
+    class general(View):  # noqa
+        infra_map = BootstrapSelect('infrastructure_mapping')
+        name = TextInput(name='name')
+        name_help_text = Text(locator='.//div[contains(@id,"name")]/span')
+        description = TextInput(name='description')
+        select_vm = RadioGroup('.//div[contains(@id,"vm_choice_radio")]')
+
+        @property
+        def is_displayed(self):
+            return self.infra_map.is_displayed and self.name.is_displayed
+
+    @View.nested
+    class vms(View):  # noqa
+        import_btn = Button('Import')
+        importcsv = Button('Import CSV')
+        hidden_field = HiddenFileInput(locator='.//input[contains(@accept,".csv")]')
+        # TODO: Replace string keys by integer keys after row indexing issue get fixed
+        # TODO: Replace Text by Button or GenericLocatorWidget after button text get added
+        table = Table('.//div[contains(@class, "container-fluid")]/table', column_widgets={
+            'Select': Checkbox(locator=".//input"),
+            1: Text('.//span/button[contains(@class,"btn btn-link")]')})
+        filter_by_dropdown = SelectorDropdown('id', 'filterFieldTypeMenu')
+        search_box = TextInput(locator='.//div[contains(@class, "modal-content")]'
+                                       '//div[contains(@class,"input-group")]/input')
+        clear_filters = Text(".//a[text()='Clear All Filters']")
+        error_text = Text('.//h3[contains(@class,"blank-slate-pf-main-action") and '
+                          'contains(text(), "Error:")]')
+        error_icon = Text(locator='.//span[contains(@class, "pficon-error-circle-o")]')
+        popover_text = Text(locator='.//div[contains(@class, "popover-content")]')
+
+        @property
+        def is_displayed(self):
+            return (self.table.is_displayed and
+                 (len(self.browser.elements(".//div[contains(@class,'spinner')]")) == 0))
+
+        def filter_by_name(self, vm_name):
+            try:
+                # remove `if` cond https://github.com/ManageIQ/manageiq-v2v/issues/771 fixed
```

Comment:
Need action:
This issue `closed` PR merged. can you just check its back ported or not?